app/src/main/java/org/jellyfin/androidtv/integration/dream/composable/DreamHost.kt

```diff
@@ -55,7 +56,7 @@ fun DreamHost() {
 
 	DreamView(
 		content = when {
-			mediaItem != null -> DreamContent.NowPlaying(mediaItem)
+			mediaItem?.mediaType == MediaType.Audio -> DreamContent.NowPlaying(mediaItem)
```

Comment:
But my audio books

Reply:
We don't currently support audiobooks in the app. When we do this will need to be updated 😉